scripts/globals/abilities/stay.lua

```diff
@@ -21,6 +25,30 @@ end;
 -- onUseAbility
 -----------------------------------
 
-function onUseAbility(player,target,ability)
-    player:petStay();
+function onUseAbility(player,target,ability,action)
+    pet = player:getPet();
```

Comment:
Ok, I’ll make it local, commit, and do a new PR.

 

Thanks!

 

From: bendangelo [mailto:notifications@github.com] 
Sent: Friday, March 4, 2016 3:48 PM
To: DarkstarProject/darkstar <darkstar@noreply.github.com>
Cc: lenonk <lenon@sanctuary.org>
Subject: Re: [darkstar] BST stay command now works per retail. The amount healed may need to be (#2849)

 

In scripts/globals/abilities/stay.lua <https://github.com/DarkstarProject/darkstar/pull/2849#discussion_r55093753> :

> @@ -21,6 +25,30 @@ end;
>  -- onUseAbility
>  -----------------------------------
>  
> -function onUseAbility(player,target,ability)
> -    player:petStay();
> +function onUseAbility(player,target,ability,action)
> +    pet = player:getPet();

Yes it has to be defined as local pet or else it's defined as a global variable. Which I think it stupid on luas part.

—
Reply to this email directly or view it on GitHub <https://github.com/DarkstarProject/darkstar/pull/2849/files#r55093753> .  <https://github.com/notifications/beacon/ABIIJTiK-9dLHCHBgZplxSdLcvDZBIG9ks5pqKidgaJpZM4HnQDO.gif>